### ISSUE TITLE
BuildError: hibp fails to build

### ISSUE THREAD

**Issue by wKovacs64** [pastelsky/bundlephobia#436]
## Package name

hibp

- https://bundlephobia.com/result?p=hibp@10.0.0
- https://github.com/wKovacs64/hibp

### Entire (stringified) error that I see in my browser console

```
{
  "error": {
    "code": "BuildError",
    "message": "Failed to build this package.",
    "details": {
      "name": "BuildError"
    }
  }
}
```

### Additional Information

I introduced an `exports` map in `package.json` in v10, however I published a couple alpha versions prior to the final 10.0.0 release and they were building on Bundlephobia just fine. I believe the only thing that changed between a version that built and one that didn't was one of my dependencies ([jssha](https://github.com/Caligatio/jsSHA)) enhanced its own `exports` map to include a module version of the file I am deep importing. All local bundling/building with various tools (tsdx, microbundle, rollup, create-react-app, etc.) succeed, but Bundlephobia seems to be having a hard time with it.

Thank you for your work!

**Comment by pastelsky** [comment]
This is failing due to Terser failing to minify certain syntax in one of the files. 

```
 name: 'SyntaxError',
    message: 'Unexpected token: keyword (extends)',
    filename: '0',
    line: 9,
    col: 6016,
    pos: 6403
```

Could you verify if all of the dist files are using valid JS syntax (ES5 / ES6) ?

**Comment by wKovacs64** [comment]
The `dist` contents are generated by Rollup, so it would surprise me if they were not valid JS syntax.

That said, upon investigating the contents and looking for the `extends` keyword (ES6 classes), I did find a problem with my build configuration not honoring my build targets, so the UMD output did contain `extends` when it shouldn't - but I wouldn't think that would cause Bundlephobia any trouble? The module-for-browsers build output still contains `extends`, but that's intentional.

I just published `hibp@10.0.1-alpha.0` with the corrected `dist` contents, but Bundlephobia still fails to build that version. Does it not support ES6 classes? Let me know if there is anything else I can do on my end to troubleshoot.

Thank you!
